custom_metrics/almanac.js

```diff
@@ -1,16 +1,6 @@
 //[almanac]
 // Uncomment the previous line for testing on webpagetest.org
 
-// README! Instructions for adding a new custom metric for the Web Almanac.
```

Comment:
I agree with this change, but just to be safe can you revert it to avoid submitting changes to it while the other PRs are in flight?